sdk/digitaltwins/Azure.DigitalTwins.Core/src/Queries/QueryBuilders/SelectQuery.cs

```diff
@@ -88,7 +88,7 @@ public FromQuery SelectOverride(string literalQuery)
         public override AdtQueryBuilder Build()
         {
             // Build can only be called on queries that have (at minimum) SELECT and FROM clauses
-            throw new InvalidOperationException("Queries that contain just a SELECT statement are not valid.");
+            throw new InvalidOperationException("Queries that contain just a SELECT must also contain a FROM clause.");
```

Comment:
`Invalid query: Missing a FROM clause.`

The sentence reads a bit strange. I think we have to revisit these error messages in one of our final PBIs. Error handling and validation.